support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -360,7 +360,7 @@ EventsAndTriggersAtSlabs:
   - Trigger:
       Slabs:
         EvenlySpaced:
-          Interval: 5
+          Interval: 1
```

Comment:
did you intend to make this change?

Reply:
yeah I've noticed that the ObservationAhA/B usually fails when this is 5

Reply:
Ok I hope this doesn't slow down the run too much. Have an idea?